portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,45 +48,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             try {
                 maxNumComments = Integer.parseInt(maxNumCommentsStr);
             } catch (NumberFormatException e) {
-                System.err.println("Could not convert to int: " + maxNumCommentsStr);
+                System.err.println("Could not convert to int: " +
+                    maxNumCommentsStr);
             }
         }
 
         String page = request.getParameter("page");
 
-        Query query = new Query("Comment-" + page).addSort("date", SortDirection.DESCENDING);
-        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+        Query query = new Query("Comment-" + page)
+            .addSort("date", SortDirection.DESCENDING);
+        DatastoreService datastore = DatastoreServiceFactory
+            .getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
 
-        comments = new ArrayList<>();
+        comments = new ArrayList < > ();
```

Comment:
That is really weird. Auto-format should never add the space to make it `< >`. Can you manually change it to `<>`?